bundle/manifests/kube-green.clusterserviceversion.yaml

```
@@ -201,7 +201,7 @@ spec:
           - subjectaccessreviews
           verbs:
           - create
-        serviceAccountName: kube-green-controller-manager
+        serviceAccountName: {{ .Values.serviceAccount.name }}
```

This file should be reverted, since it is used by olm and it is outside the chart folder

```suggestion
        serviceAccountName: kube-green-controller-manager
```